packages/cache/src/resource-manager.ts

```diff
@@ -200,17 +233,28 @@ export class ResourceManager {
     }
   }
 
-  private async fetchCode(
-    url: string,
-    options: RequestInit = {}
-  ): Promise<any> {
+  private async fetchCode(url: string): Promise<any> {
     try {
-      const response = await fetch(url);
+      const response = await fetch(url, {
+        headers: {
+          Accept: 'application/json',
+          'Content-Type': 'application/json',
```

Comment:
AFAICT this is a GET request so we shouldn't need to pass a content type since we have not body